lib/components/DevCards.js

```diff
@@ -1,10 +1,17 @@
-var objectAssign = require('object-assign');
+var createHistory = require('history').createHashHistory;
+var router = require('../router');
 
 var React = require('react');
 
 var $ = React.createElement;
 
-var DevCard = require('./DevCard');
+var utils = require('../utils');
+var find = utils.find;
+
+var history = typeof window !== 'undefined' && createHistory({queryKey: false});
```

Comment:
This is a possibility. The thing is, with HMR and all, the component might unmount and remount and I am not sure if this could be a problem.